### PR TITLE
Meteor 3 migration

### PR DESCRIPTION
**Pull request by harryadel**
Add changes from @activitree [fork](https://github.com/activitree/scss3).

There were lots of style changes so I tried to distill them to the absolute minimum. 

cc @StorytellerCZ @jankapunkt

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -7,17 +7,20 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
-  use: ['caching-compiler@1.2.2 || 2.0.0', 'ecmascript@0.15.1'],
+  use: [
+    'caching-compiler@2.0.0-alpha300.15',
```
Comment (ebroder) [inline package.js:11]: I think this should still be `||`'d with 1.2 so it continues to work with Meteor 2

**Comment by dallman2** [comment]
Hey all,

I recently discovered that the css module [package](https://github.com/akanix42/meteor-css-modules) i was using is both broken and unmaintained. I am trying to move my project over to using this PR (thanks @harryadel). However, when i start my app, I only see a white screen when navigating to my app. No errors in the console, server or client side...

What sort of configuration is needed in general to get this package to work?

**Comment by harryadel** [comment]
@dallman2 Nothing novel, pretty much what was said [here](https://github.com/Meteor-Community-Packages/meteor-scss#usage). I guess things aren't working because they're setup to work with https://github.com/akanix42/meteor-css-modules

**Comment by dallman2** [comment]
@harryadel disregard my issue! I was confusing this package for a css module compiler. From what I understand now, most of the community has moved to vite for their module compiler.

**Comment by StorytellerCZ** [comment]
@harryadel I think we should update versions and release a beta for people who need it. Any other updates for it would be highly appreciated.

File: package.js [hunk 2]
```diff
@@ -7,17 +7,20 @@ Package.describe({
...
   npmDependencies: {
-    'node-sass': '4.14.1',
-  },
-});
-
-Package.onUse(function(api) {
-  api.versionsFrom('2.3');
-  api.use('isobuild:compiler-plugin@1.0.0');
-});
+    sass: '1.69.5'
```
Comment (StorytellerCZ) [inline package.js:16]: Do we want to go to the `sass` package? Wasn't there in the past migration to `node-sass`?

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: package.js (hunk 2 above)
Comment (jankapunkt) [reply package.js:16]: @StorytellerCZ I totally don't know at this point

**Comment by harryadel** [comment]
Okidoki, I've been putting this. Thank you guys for getting things moving again @StorytellerCZ @jankapunkt. But I'd rather close this PR as per my [comment here](https://github.com/meteor/meteor/issues/13119#issuecomment-2100098022), activity tree fork isn't fully backwards compatible; meteor imports don't work. So, I'll be closing this PR in favor of another I'm going to start now.

**Pull request closed without merging**